<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddCompartmentDialog</class>
 <widget class="QDialog" name="AddCompartmentDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>174</width>
    <height>95</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="abbreviationLabel">
       <property name="text">
        <string>Abbreviation:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="abbreviationInput"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="nameLabel">
       <property name="text">
        <string>Name:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="nameInput"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>AddCompartmentDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>164</x>
     <y>77</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>94</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>AddCompartmentDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>164</x>
     <y>83</y>
    </hint>
    <hint type="destinationlabel">
     <x>173</x>
     <y>94</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>abbreviationInput</sender>
   <signal>textChanged(QString)</signal>
   <receiver>AddCompartmentDialog</receiver>
   <slot>activateButton()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>127</x>
     <y>19</y>
    </hint>
    <hint type="destinationlabel">
     <x>220</x>
     <y>37</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>nameInput</sender>
   <signal>textChanged(QString)</signal>
   <receiver>AddCompartmentDialog</receiver>
   <slot>activateButton()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>130</x>
     <y>48</y>
    </hint>
    <hint type="destinationlabel">
     <x>242</x>
     <y>58</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>activateButton()</slot>
  <slot>check_abbreviation()</slot>
 </slots>
</ui>
